neuralprophet/forecaster.py

```diff
@@ -2518,8 +2585,16 @@ def _reshape_raw_predictions_to_forecst_df(self, df, predicted, components):
                 where yhat<i> refers to the i-step-ahead prediction for this row's datetime.
                 e.g. yhat3 is the prediction for this datetime, predicted 3 steps ago, "3 steps old".
         """
-        if isinstance(df, dict):
-            raise ValueError("Received more than one DataFrame. Use a for loop for many dataframes.")
```

Comment:
maybe don't say " for loop"
but just say 
"Received a dictionary instead of a single DataFrame."